flag/service/azure/hostcluster/tenant/tenant.go

```diff
@@ -0,0 +1,7 @@
+package tenant
+
+type Tenant struct {
```

Comment:
Just a silly thing. I don't think "subscription" is a field of "Tenant" in the azure naming.
They kinda have a 1:n relationship.
But guess it doesn't make confusion, we can leave it like it is I guess.

Reply:
Correct. I didn't know which name to use. I needed a term that groups together tenantid, subscriptionid and partnerid. I thought about a GiantSwarm tenant, and I chose that. But I'm not sure.

Reply:
Something related to an `organization`? Like `OrganizationAzureSettings`?

Reply:
It's used for the control plane, so I really don't like organization.

Reply:
ok Tenant is good enough, let's not overcomplicate things